Detection[] detections
int32 w_used
int32 h_used
sensor_msgs/CameraInfo camera_info
time stamp
